### ISSUE TITLE
How to use custom yolov3 models on dlstreamer

### ISSUE THREAD

**Issue by MohammadsAdrA-AI** [dlstreamer/dlstreamer#298]
I have a yolov3 model which has 3 classes. I trained that in PyTorch, then converted to onnx, then converted to IR openvino by openvino model optimizer toolkit. I tried to create a model-proc JSON file, I'm not sure if it's correct or not. but I am thinking that's the issue. 
you can see the JSON file and the code that I am running, and the error in the following: 
model_proc.json
{
    "json_schema_version": "2.1.0",
    "input_preproc": [],
    "output_postproc": [
        {
            "layer_name": "output",
            "converter": "tensor_to_bbox_yolo_v3",
            "anchors": [
                10.0,
                13.0,
                16.0,
                30.0,
                33.0,
                23.0,
                30.0,
                61.0,
                62.0,
                45.0,
                59.0,
                119.0,
                116.0,
                90.0,
                156.0,
                198.0,
                373.0,
                326.0
            ],
            "masks": [
                6,
                7,
                8,
                3,
                4,
                5,
                0,
                1,
                2
            ],
            "iou_threshold": 0.4,
            "classes": 3,
            "labels": [
                "car", "person", "truck"
            ],
            "bbox_number_on_cell": 3,
            "cells_number":19,
            "entry_points": ["detector/yolo-v3/Reshape", "detector/yolo-v3/Reshape_285", "detector/yolo-v3/Reshape_304"]
        }
    ]
}

the code that I am running for the inference
GST_DEBUG=3 gst-launch-1.0 -e filesrc location="$VIDEO" ! \
  qtdemux ! \
  avdec_h264 ! \
  gvadetect model="${DETECTION_MODEL}" device=CPU threshold="$THRESHOLD" inference-interval="$INFINT" model-proc="$JSON_PATH" ! \
  gvatrack tracking-type=short-term ! \
  gvawatermark ! \
  autovideosink 
the erro said the cells_number is invalid 

P.S I used this repo for custom training: 
https://github.com/ultralytics/yolov3

**Comment by MohammadsAdrA-AI** [comment]
@dsmertin Thanks for your answer, I could fix the cells_number but I still have issue with post processing. is there anyway that I create the mode-proc file, not manually?

**Comment by akwrobel** [comment]
Hi @MohammadsAdrA-AI  there is currently no support for automated model-proc file creation.

**Comment by akwrobel** [comment]
This issue has been answered, closed due to age
